lib/cpython/build/bin/cpython-cc

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env sh
+set -e
+
+SYSROOT=$(eval echo ~/.symbolica)
+
+exec ~/.symbolica/bin/cc -I$SYSROOT/include/python3.8 -DCPYTHON_HOME=L\"$SYSROOT\" -DCPYTHON_EXEC=L\"$SYSROOT/bin/python3\" -DCPYTHON_PATH=L\"$SYSROOT/lib/python3.8\" "$@" -lpython3.8
```

Comment:
"${SYSROOT}/bin/cc"?

Reply:
I've been wondering more generally whether we expect the user to set $SYMBOLICA_HOME in their environment and then use that throughout these scripts. Certainly makes things easier in Docker land where we can set it inside the container when we build it. I think other advantages would be that it would potentially allow side-by-side installs of different versions on a system.